File: official/docs/go/referral/update-referral-user.go
package example

import (
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("<YOUR_TEST/PRODUCTION_API_KEY>")
	client := easypost.New(apiKey)

	_, _ = client.UpdateReferralCustomerEmail("user_...", "new_email@example.com")
}

File: official/docs/python/referral/update_referral_user.py
import os

import easypost


easypost.api_key = os.getenv("EASYPOST_API_KEY")

updated_referral_user = easypost.Referral.update_email(
    "new_email@example.com",
    "user_...",
)

print(updated_referral_user)
